docs/migrate_to_v2.md

```diff
@@ -28,33 +28,41 @@ const auth = function () {
 
 #### v2 Authentication: Credentials object
 
-```javascript
-{ 
+```typescript
+import { Credentials } from 'skyflow-node';
+
+const credentials: Credentials = { 
     apiKey: "<YOUR_SKYFLOW_API_KEY>"
-}
+};
 ```
 
 // Option 2: Environment Variables (Recommended)
 // Set SKYFLOW_CREDENTIALS in your environment
 
 // Option 3: Credentials File
 
-```js
-const credentials = { path: "<YOUR_CREDENTIALS_FILE_PATH>" }; // Replace with the path to credentials file
+```typescript
+import { Credentials } from 'skyflow-node';
+
+const credentials: Credentials = { path: "<YOUR_CREDENTIALS_FILE_PATH>" };
```

Comment:
Same feedback here. Not needed and adds confusion.

Citing the readme readme 🤓 

> No matter how your project runs, however, stop once the project works once. Extended usage instructions belong in dedicated documentation files, not your README.